app/web/src/store/workspaces.store.ts

```diff
@@ -181,6 +234,7 @@ export const useWorkspacesStore = () => {
         watch(
           () => this.selectedWorkspacePk,
           () => {
+            this.GET_INTEGRATIONS();
```

Comment:
If I'm reading this right, when this route is run, then the below code will execute in this watch?